### PR TITLE
Added Privacy Notice

### PR DESCRIPTION
**Pull request by Cavalier07**
## Description and relevant Jira numbers
- Added privacy notice component 
- Fixed issue with styling on search result (as @buzzub solved my issue with displaying the left hand bars

## Pull Request checklist:

- [ ] [WIP] tag removed from PR title
- [ ] PR has an understandable description

## Git feature branch checklist:

- [ ] branch name comply with our branching strategy
- [ ] git branch contains relevant JIRA ticket number
- [ ] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [ ] Code has been tested manually
- [ ] Tests are passing
- [ ] PR link added to JIRA
- [ ] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

IMG_135C74D0662F-1

IMG_453D812A16F4-1

### CODE REVIEW

File: src/pages/test-results-search/components/search-result/search-result.scss [hunk 1]
```diff
@@ -3,12 +3,7 @@ search-result {
     margin-left: 0;
     margin-right: 0;
     width: 100%;
-  }
-  .indicator {
-    background: rgba(155, 155, 155, 0.5);
-    position: absolute;
-    height: 120px;
-    width: 8px;
+    border-left: 8px solid rgba(87, 73, 73, 0.5);
```
Comment (buzzub) [inline src/pages/test-results-search/components/search-result/search-result.scss]: Should this be a standard colour or declared in the variables mixin?

File: src/pages/communication-page/components/privacy-notice/privacy-notice.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+privacy-notice {
+  .container {
+    margin: 40px 32px 0px 32px;
+    background-color: #f3f3f3;
```
Comment (buzzub) [inline src/pages/communication-page/components/privacy-notice/privacy-notice.scss]: this should probably be `gds-grey-4` or `light` from standard colours

File: src/pages/test-results-search/components/search-result/search-result.scss (hunk 1 above)
Comment (buzzub) [reply src/pages/test-results-search/components/search-result/search-result.scss]: This is `gds-black`, so could be switched to that instead.

File: src/pages/communication-page/components/privacy-notice/privacy-notice.scss (hunk 2 above)
Comment (Cavalier07) [reply src/pages/communication-page/components/privacy-notice/privacy-notice.scss]: I can only go with the shade the design requested , will check with Nad when he's back in and working and change if needed

File: src/pages/test-results-search/components/search-result/search-result.scss (hunk 1 above)
Comment (Cavalier07) [reply src/pages/test-results-search/components/search-result/search-result.scss]: Have updated this to the hex version , which isn't one of our standard colours will check with Nad when he's back in and see if he's happy for us to use one of them

File: src/pages/communication-page/components/privacy-notice/privacy-notice.scss (hunk 2 above)
Comment (buzzub) [reply src/pages/communication-page/components/privacy-notice/privacy-notice.scss]: If you can spot the difference between #f3f3f3 and #f4f4f4 I'll give you £100 ;)

**Pull request merged**